content/crutchcorn/collections/art-of-accessibility/posts/art-of-a11y-text/index.md

```diff
@@ -0,0 +1,561 @@
+
+-----
+
+# Headings
+
+When talking about text, it's only natural that heading would come into the conversation.
+
+In HTML, we have 6 levels of headings:
+
+- <p class="text-style-headline-1"><code>h1</code></p>
+- <p class="text-style-headline-2"><code>h2</code></p>
+- <p class="text-style-headline-3"><code>h3</code></p>
+- <p class="text-style-headline-4"><code>h4</code></p>
+- <p class="text-style-headline-5"><code>h5</code></p>
+- <p class="text-style-headline-6"><code>h6</code></p>
+
+Each of these heading levels represents a new segment of the site with information to all users about how to scan and navigate a page's contents.
+
+While headings might seem fairly straightforward at first, they have a bit more depth than you might expect at first.
+
+## Screen Readers
+
+Headings help structure the page for your users. Not only can they provide visual structure to a page with the font size, but they can report to a scree-reader to help quickly navigate through a page as well.
+
+Here's one such example of a screen reader - Voiceover built into macOS - navigating [our home page](/) via their "rotor" feature, which lists all headings:
+
+<video src="./macos_voiceover_heading_navigation.mp4" title="The macOS voiceover shows a list of headings to the user and allows them to quickly navigate a table of headings"></video>
+
+### Screen Reader Cheat Sheet
+
+Here's a list of screen reader commands related to headings in [Voiceover](https://support.apple.com/guide/voiceover/welcome/mac), [NVDA](https://www.nvaccess.org/download/), and [JAWS](https://www.freedomscientific.com/products/software/jaws/):
+
+| Screen Reader | Command                                                      | Shortcut                                                     |
+| :------------ | :----------------------------------------------------------- | ------------------------------------------------------------ |
+| Voiceover     | Show heading list (via the Voiceover rotor in the demo above) | <kbd>Command (⌘)</kbd> + <kbd>Ctrl</kbd> + <kbd>U</kbd>      |
+| Voiceover     | Go to next heading                                           | <kbd>Command (⌘)</kbd> + <kbd>Ctrl</kbd> + <kbd>H</kbd>      |
+| Voiceover     | Go to previous heading                                       | <kbd>Command (⌘)</kbd> + <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>H</kbd> |
+| NVDA          | Go to next heading                                           | <kbd>H</kbd>                                                 |
+| NVDA          | Go to next heading of level [1-6]                            | <kbd>1</kbd> - <kbd>6</kbd>                                  |
+| NVDA          | List all headings                                            | <kbd>Insert</kbd> + <kbd>F7</kbd>                            |
```

Comment:
Consider adding <kbd>d</kbd> to jump landmarks for NVDA, which would give more context to your future discussion of landmarks

Reply:
I probably won't do this, only because this section is about headings specifically. Each chapter should list their own screen-reader table like this